packages/office-ui-fabric-react/src/components/GroupedList/examples/GroupedList.Basic.Example.tsx

```diff
@@ -86,6 +86,7 @@ export class GroupedListBasicExample extends React.Component<{}, IGroupedListExa
         selection={this._selection}
         selectionMode={SelectionMode.multiple}
         compact={this.state.isCompactMode}
+        styles={{ root: { width: '100%' } }} // expand the row all the way so it matches to the GroupHeader
```

Comment:
I've actually tried to tackle this in the past re: the default case. You can see this problem on SharePoint document libraries... #6705

Reply:
So what was the reason the PR was not merged? I read all the comments but did not get why this was not a solution. Also, my opinion is that we should not do any changes in the source default styles of the `DetailsRow` to accommodate its use in the `GroupedList` example. When we use `GroupedList` outside the context of `DetailsList` and we just happened to use a `DetailsRow` as a renderer of the data cells we should just customize the styles of it to properly align with its headers. `DetailsRow` is a subcomponent of `DetailsList` and it's styled for its use within it. If you see this [example](https://developer.microsoft.com/en-us/fabric#/components/detailslist/grouped) you can see that the issue is not present as it has the necessary `viewport` value passed from its parent `DetailsList`.

So in short, if we decided to use `DetailsRow` outside of its initial intend we should provide custom styles so it can play well with the new parent. `DetailsRow` should not care or know that it's used within `GroupedList`. Let me know if my rationale is missing something and I am wrong on this one.

Reply:
Yea, it has to do with how the viewport is used which _does_ impact DetailsList in some cases. We can fix the example for the time-being, but I think this should be a default of the control. However, as I found out with the above PR there are a few edge cases to consider.